### ISSUE TITLE
Checksum mismatch for 3.9 Team Edition

### ISSUE THREAD

**Issue by dmke** [mattermost/docs#1190]
The SHA256 checksum committed in c3eb92cec964fd793a5e68183eb146f4f57e8946 does not reflect the actual checksum of the file:

```
$ sha256sum mattermost-*.tar.gz
c6179f7b2282cfbc8f0a691a90b41b554b62726f1dfb036fc59eed635556c8d9  mattermost-team-3.9.0-linux-amd64.tar.gz
```

I've downloaded the archive from different machines on different networks to eliminate a MITM, but to no avail (the SSL certifcate looks OK, and each time it yielded the `c6179...` checksum).

**Comment by JeffSchering** [comment]
Thanks @dmke! I'm looking into it now.

**Comment by dmke** [comment]
FYI:

```
$ sha256sum bin/platform 
283f4d644a41c60e671b79e2cf5343eaed663cd0aaad47949754595200bb4344  bin/platform
```

**Comment by JeffSchering** [comment]
Thanks @dmke 

Looks like the wrong number was published. See PR https://github.com/mattermost/docs/pull/1191 for the fix.

**Comment by dmke** [comment]
Thanks!

**Comment by dmke** [comment]
Do you, by any chance, have a machine readable version of the release archive?

I find it hard to follow releases and currently am scraping https://docs.mattermost.com/administration/upgrade.html to check if I missed a patch release (the Mattermost Newsletter only covers minor releases).

A JSON file with the latest releases like this would be very helpful:

```json
[
  {
    "edition": "enterprise",
    "version": "3.9.0",
    "url": "https://releases.mattermost.com/...",
    "sha256sum": "..."
  },{
    "edition": "team",
    "version": "3.9.0",
    "url": "https://releases.mattermost.com/...",
    "sha256sum": "..."
  }
]
```

Who knows, maybe you could even autogenerate the `upgrade.rst` from that file?

**Comment by JeffSchering** [comment]
That's an interesting idea... I'll mention it to the product managers to see what they think.

**Comment by dmke** [comment]
@JeffSchering, is there news regarding the JSON feed?

After the recent move of the content from `upgrade.rst` to `version-archive.rst`, my ["mattercheck"](https://github.com/dmke/mattercheck) helper became defunct. While this was an easy fix, I need to depend on the URL and the document structure for it to work properly...

If you need another spin for the product managers: Mattermost Server instances could query the feeds by themselves and display a banner to admins informing them about a new Major/Minor/Patch release. Not only would that eleminate my helper, but also add value to your product :-)

**Comment by JeffSchering** [comment]
Hi @dmke , I'm not sure what's going on now... I haven't been involved much with Mattermost for the last couple of months. If you'd like to push the JSON feed idea forward, you can take it up directly with @lfbrock , @jasonblais or @esethna on https://pre-release.mattermost.com/core/channels/product-management, or you could make the suggestion at https://mattermost.uservoice.com
